### DISCUSSION TITLE
Curious

### DISCUSSION THREAD

**Discussion by Grizinator1** [community/community#56272] [New to GitHub]
### Select Topic Area

General

### Body

What is a repository?

**Comment by obiwac** [comment]
It's the place where you store all the sources (i.e. code, assets, ...) for your project, on a remote (such as GitHub) or locally.

**Comment by radar-blip** [comment]
Info: [Github Repositories](https://docs.github.com/en/repositories/creating-and-managing-repositories/about-repositories)

**Comment by airtower-luna** [comment]
Check the Git Book for what Git and repositories are: https://git-scm.com/book/en/v2 The general idea is explained in the first half of the first chapter, chapters 2 and 3 teach you the basics of how to work with them.
